pkg/pillar/cmd/zedrouter/flowstats.go

```diff
@@ -54,17 +54,27 @@ type aclAttr struct {
 	aclName   string
 	chainName string
 	action    string
-	vif       string
 	bridge    string
-	netUUID   uuid.UUID
+	intfname  string // App virtual interface name assigned by cloud template
+}
+
+type bridgeAttr struct {
+	bridge  string
+	netUUID uuid.UUID
+}
+
+type appInfo struct {
+	ipaddr    net.IP
+	intf      string
+	localintf string
 }
 
 type networkAttrs struct {
-	ipaclattr  map[int]map[int]aclAttr // appNum, ACLNum, acl attributes
-	appIPAddrs map[int][]net.IP        // appNum, IP addresses (may belong to diff bridges)
-	intfAddrs  []net.Addr              // device interface addresses
-	bnNet      map[string]aclAttr      // mainly need to range all the bridge interfaces
-	appNet     map[int]uuid.UUID       // max 256 apps
+	ipaclattr map[int]map[int]aclAttr // appNum, ACLNum, acl attributes
+	appIPinfo map[int][]appInfo       // appNum, IP addresses/intfs (may belong to diff bridges)
+	intfAddrs []net.Addr              // device interface addresses
+	bnNet     map[string]bridgeAttr   // mainly need to range all the bridge interfaces
+	appNet    map[int]uuid.UUID       // max 256 apps
```

Comment:
An app can have multiple network interfaces each connected to a different network instance. Shouldn't this be a map of maps?